.circleci/config.yml

```diff
@@ -2003,15 +1745,12 @@ jobs:
       workflow-name:
         type: string
         default: ""
-      test-name:
-        type: string
-        default: "GCP-Daily-Services-Crypto-Update-5N-1C.json"
     executor:
       name: build-executor
       workflow-name: << parameters.workflow-name >>
     steps:
       - run-jrs-experiment:
-          experiment-name: << parameters.test-name >>
+          experiment-name: << parameters.workflow-name >>.json
```

Comment:
Should change the common step `run-jrs-experiment` instead of changing all of the calls to this step.

Reply:
Fixed. Thanks Quan !